app/redis_connection.py

```diff
@@ -0,0 +1,10 @@
+import redis
+import os
+
+redis_conn = None
+if os.environ["REDIS_CREATE"] == "true":
```

Comment:
On environments where we do not create redis, we still want to do this, no?

Reply:
We would just connect to an aws hosted redis instead

Reply:
Probably for most project that would be the case, yes. Should we show how to do that in the demo as well? I don't know how to do that haven't been able to find any examples of it in our other repos.

Reply:
We should probably eventually have scripts that can create redis on aws for non dev/local envs (some future PR), I think cortado/universal status have some similar examples of this (spinning up dynamodb/elasticsearch on aws). I just think we probably don't need this if, since eventually we should have external redis's on non dev/local envs.

Reply:
I see. I'll leave a TODO comment there but I think we should leave this if statement in for now since it will break if REDIS_CREATE is set to false. We can change it in the future PR that creates redis on aws for non dev/local envs.